book/HAS/hybrid-application-service-design.md

```diff
@@ -0,0 +1,88 @@
+# Hybrid Application Service (HAS) Design
+
+## Application CR Create
+
+HAC collects the information required for creating applications. An application is a collection of one or more components. Required information to be collected can be found in the API [doc](). When all the application information is available, HAC will create the Application CR to trigger the application creation. The HAS Application CR Operator will:
+1. Validate all the required fields in the Application CR
+2. If the app model repository and the GitOps repository repos are specified:
+   1. It will validate if the specified repo exists.
```

Comment:
Will users see an error if the specified repos do not exist?

Reply:
I just checked the application controller, and the application controller doesnt error out if the specified repo doesnt exists. Just assumes it does.

I just changed this sentence in question to `It will be used for the Application model and GitOps repository.`

Bug logged https://issues.redhat.com/browse/DEVHAS-538